README.md

```diff
@@ -5,6 +5,28 @@
 
 ## Build Instructions
 
+### Condensed Steps
+
+Please refer to the sections below for more detailed information.
+
+1. Download and install Xcode.
+1. `git clone git@github.com:wordpress-mobile/WordPress-iOS.git` in the folder of your preference.
+1. `cd WordPress-iOS` to enter the working directory.
+1. `rake dependencies` this may take some time to complete.
```

Comment:
I wonder if we should explain what `rake dependencies` does. 

```
`rake dependencies` to install all dependencies required to run the project.
```

Reply:
I like it but I also intentionally tried to keep each line VERY short and I still wanted to keep in the "this may take some time to complete" part because it's disconcerting when quite a lot of time goes by and you're not sure whether something was working or not. I'll add both for now, and I think I may move the timing part to lower in the document on the next pass.

Reply:
That's true. I was concerned about making the line longer too. I wanted to emphasize that we use `rake dependencies` to install all dependencies for developers who already know about `pod install` and other types of dependency managers. But since we already explain this in the _Third party tools_ section, maybe it's not needed. 🤷‍♂ I leave that up to you. 🙂

Reply:
Sweet! I'm keeping it for now and will likely move it in the next edit.